components/home/headerBanner.tsx

```diff
@@ -5,8 +5,10 @@ const HeaderBanner: FunctionComponent = () => {
   return (
     <img
       alt="Header banner"
-      src="/visuals/home/headerBanner.webp"
```

Comment:
If you add a new headerImage.webp file, then delete the headerBanner.webp file as it's not used anymore